frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/YarnInstaller.java

```diff
@@ -14,6 +14,9 @@ public class YarnInstaller {
 
     public static final String INSTALL_PATH = "/node/yarn";
 
+    public static final String ATLASSIAN_YARN_DOWNLOAD_ROOT =
```

Comment:
maybe we could group all the ATLASSIAN_... constants in some additional java file and just reference as needed?